content/02.introduction.md

```diff
@@ -90,12 +90,25 @@ Coronaviruses are known to cause respiratory illnesses in humans through the fol
 The symptoms of infection by SARS-CoV-2 can vary greatly, making it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate infection and should prompt isolation.
-[@doi:10/ggnxb3] found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment scores, as well as high levels of d-dimer.
-Mortality might be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing.
-They also found that survivors continued to shed the virus for a median of 20 days and a maximum of at least 37 days.
+[@doi:10/ggnxb3] found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment scores, as well as high levels of d-dimer. 
+Mortality might be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing. 
+[@doi:10/ggnxb3] reported that survivors continued to shed the virus for a median of 20 days and a maximum of at least 37 days.
+A later study reported that many hospitalized patients had radiographic findings such as ground-glass opacity and bilateral patchy shadowing, and most had lymphocytopenia. [@doi:10.1056/NEJMoa2002032]
+COVID-19 can affect diverse body systems in addition to causing respiratory problems.  
+For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions [@doi:10.1016/j.kint.2020.05.006].
+COVID-19 has also been associated with an increased incidence of large vessel stroke, particularly in patients under the age of 40 [@doi:10.1056/NEJMc2009787], and other thrombotic events including pulmonary embolism and deep vein thrombosis [@doi:10.1016/j.thromres.2020.04.013].
+The mechanism behind these complications has been suggested to be related to coagulopathy with reports of antiphospholipid antidbodies present [@doi:10.1056/NEJMc2007575] and elevated levels of d-dimer and fibrinogen degradation products (FDP) elevated in deceased patients [@doi:10.1111/jth.14768].
+Other viral infections have been associated with coagulation defects, notably the coronavirus Severe Acute Respiratory Syndrome (SARS) which was also noted to lead to disseminated intravascular coagulation (DIC) and was associated with both pulmonary embolism and deep vein thrombosis [@doi:10.1002/jmv.23354].
+A wide range of viral infections can effect the coagulation cascade.
+The mechanism behind these insults has been suggested to be related to inflammation-induced increases in von Willebrand factor, leading to a pro-coagulative state [@doi:10.1002/jmv.23354].
 
```

Comment:
```suggestion
Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].  
Once hospitalized, outcomes have generally been poor with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032] [@doi:10.1001/jama.2020.2648] [@doi:10.1001/jama.2020.4031].

```

Reply:
Should I go ahead and talk about mortality here or is that too much?   We could end up pulling in lots of different mortality statistics based on differing populations (ventilated patients vs. all hospitalized vs. "critically ill") so I want to make sure we're as clear as possible.